ro-base.obo

```diff
@@ -6162,6 +6108,26 @@ range: UBERON:0000061
 is_a: RO:0002178 ! supplies
 is_a: RO:0002179 ! drains
 
+[Typedef]
+id: RO:0020202
+name: has numerator
+def: "A relationship between a ratio or proportion and its dividend." [] {http://purl.org/dc/terms/contributor="https://orcid.org/0000-0002-8280-0487"}
+synonym: "has dividend" EXACT []
+domain: PATO:0001470
+range: PATO:0000001
+is_a: BFO:0000051 ! has part
+created_by: https://orcid.org/0000-0001-8314-2140
+
+[Typedef]
+id: RO:0020203
+name: has denominator
+def: "A relationship between a ratio or proportion and its divisor." [] {http://purl.org/dc/terms/contributor="https://orcid.org/0000-0002-8280-0487"}
```

Comment:
again, wrong property for defs, should be xref

Reply:
done